src/components/Site/Site.jsx

```diff
@@ -62,16 +62,16 @@ class Site extends React.Component {
           toggleSidebar={ this._toggleSidebar }
           links={[
             {
-              title: 'Documentation',
+              content: 'Documentation',
               url: '/concepts',
```

Comment:
We need to rethink how URLs are formed, when trying to build the site to work under a URL I had to do a lot of changes (https://github.com/webpack/webpack.js.org/tree/fix/v3-docs), we need to work on it since now in order to be able to support multiple versions in the future

Reply:
Yeah that's definitely something we need to tackle _at some point_, but is that something that needs to happen before `rebuild` is merged? Will these routes conflict with the `/v3` route you created?

Personally I think figure out version management as a follow up after finishing `rebuild` (though I understand we did need to get something up quickly before for v3 when deploying v4).

Reply:
this is indeed a pain point for community, though lets try to keep it to the plan and get `rebuild` into master asap before diving into much detail on doc versioning

Reply:
No, after merging `rebuild` is fine, would be better to do it before so we can launch new website with multiple versions support, that would make headlines! (jk).

I have tried to push that change, without success https://github.com/webpack/webpack.js.org/pull/1992 :/